README.md

```diff
@@ -22,14 +22,6 @@ Additional description about the project and its features.
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
 ### Setup
 **N/A
```

Comment:
- [ ] I think adding directives on how to obtain a local copy of your project and eventual use and run it is very necessary and makes your repo more professional and informative. Kindly add this very important section of the README file.